robot/README.md

```diff
@@ -0,0 +1,12 @@
+# Noboty Practice Bot Code
+
+> Code for the 2018 practice bot
+
+## Noboty?
+Noboty is a play on words from Team 118's Everybot. We called it this because nobody will use it.
+
+## Controls
+We used a one-joystick control system for sake of convenience. Moving the joystick on the Y axis moves the robot forwards or backwards, and the X axis causes it to rotate.
+
+### Sarah mode
+This mode cuts the speed of the robot in half, for precision movement in confined areas as well as a simple safety precaution for inexperienced drivers. Sarah mode was created when Sarah, a member of our team, smashed the robot into a wall at full speed. It is activated by pressing the red button on the joystick.
```

Comment:
Add `Author` and `License` sections at the bottom, like we do in other repositories.